src/components/CheckboxList/CheckboxList.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+import {
+    ChooseDayList,
+    ChooseDayItemLabel,
+    ChooseDayItem
+} from "./CheckboxList.style";
+
+import CheckedMark from "../../assets/check-mark.svg";
+
+const CheckboxList = () => {
+    return <ChooseDayList>
+        <ChooseDayItemLabel htmlFor='mon'>
+            <ChooseDayItem
+                type='checkbox'
+                id='mon'
+                value='1'
+                />
+            Mon
```

Comment:
To avoid copy&paste you could use cycle for this elements. Pls create a config and use array.map for it